### PR TITLE
Stop/start instances by label instead of name

### PR DESCRIPTION
**Pull request by jpatokal**
Update code to match changes to [Scheduling Compute Instances with Cloud Scheduler](https://cloud.google.com/scheduler/docs/start-and-stop-compute-engine-instances-on-a-schedule
)

### CODE REVIEW

File: functions/scheduleinstance/test/index.test.js [hunk 1]
```diff
@@ -117,7 +131,7 @@ it('startInstancePubSub: should fail with empty event payload', () => {
 
 /** Tests for stopInstancePubSub */
 
-it('stopInstancePubSub: should accept JSON-formatted event payload', async () => {
+it('stopInstancePubSub: should accept JSON-formatted event payloa with instance', async () => {
```
Comment (ace-n) [inline functions/scheduleinstance/test/index.test.js]: Typo: payloa**d**

File: functions/scheduleinstance/index.js [hunk 2]
```diff
@@ -81,25 +94,34 @@ exports.stopInstancePubSub = (event, callback) => {
...
-        // Operation complete. Instance successfully stopped.
-        const message = 'Successfully stopped instance ' + payload.instance;
-        console.log(message);
-        callback(null, message);
-      })
-      .catch(err => {
-        console.log(err);
-        callback(err);
+    const options = {
+      filter: 'labels.' + payload.label,
```
Comment (ace-n) [inline functions/scheduleinstance/index.js]: Use [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)? e.g.
```
filter: `labels.${payload.label}`
```

Also, can this variable declaration be compressed into one line?

File: functions/scheduleinstance/index.js [hunk 3]
```diff
@@ -25,37 +25,47 @@ const compute = new Compute();
...
-        // Operation complete. Instance successfully started.
-        const message = 'Successfully started instance ' + payload.instance;
-        console.log(message);
-        callback(null, message);
-      })
-      .catch(err => {
-        console.log(err);
-        callback(err);
+    const options = {
+      filter: 'labels.' + payload.label,
```
Comment (ace-n) [inline functions/scheduleinstance/index.js]: Ditto to my other comment - template strings, and compress to one line?

File: functions/scheduleinstance/test/index.test.js (hunk 1 above)
Comment (jpatokal) [reply functions/scheduleinstance/test/index.test.js]: Fixed.

File: functions/scheduleinstance/index.js (hunk 2 above)
Comment (jpatokal) [reply functions/scheduleinstance/index.js]: Done.

File: functions/scheduleinstance/index.js (hunk 3 above)
Comment (jpatokal) [reply functions/scheduleinstance/index.js]: Done.

**Pull request merged**